examples/zenohcxx/universal/z_pub_delete.cxx

```diff
@@ -33,6 +33,7 @@ int _main(int argc, char **argv) {
             exit(-1);
         }
     }
+#endif
 
```

Comment:
Does the binding over Zenoh-Pico implementing the config helpers?

Reply:
ZENOHCXX_ZENOHC or ZENOHCXX_ZENOHPICO defined in CMakeLists.txt for corresponding dependencies, user don't have to define them manually